src/main/java/edu/umich/ctools/sectionsUtilityTool/CanvasAccountAdminFinder.java

```diff
@@ -0,0 +1,95 @@
+package edu.umich.ctools.sectionsUtilityTool;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.http.client.methods.HttpGet;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * Created by pushyami on 4/15/17.
+ */
+public class CanvasAccountAdminFinder {
+
+	private static Log M_log = LogFactory.getLog(SectionsUtilityToolServlet.class);
+
+	protected boolean isAccountAdmin(HttpServletRequest request) {
+		M_log.debug("isAdmin() call");
+		String courseId = request.getParameter(Utils.LTI_PARAM_CANVAS_COURSE_ID);
+		int userId = Integer.parseInt(request.getParameter(Utils.LTI_PARAM_CANVAS_USER_ID));
+
+		int courseAccountId = getCourseAccountId(courseId);
+		if (courseAccountId == 0) {
+			return false;
+		}
+		M_log.debug("*** Course Account ID " + courseAccountId);
+		return determineIfUserIsAdminInHierarchy(userId, courseAccountId);
+	}
+
+	private boolean determineIfUserIsAdminInHierarchy(int userCanvasId, int courseAccountId) {
+		M_log.debug("determineIfUserIsAdminInHierarchy() call");
+		String url = Utils.urlConstructor("/accounts/", String.valueOf(courseAccountId),
```

Comment:
Consider making all the constant strings in url construction into constants.  That should make moving to a new version of the API much easier and setting up any new urls easier.